pkgs/by-name/to/toml11/package.nix

```
@@ -1,25 +1,37 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, cmake
+{
+  lib,
```

```suggestion
  lib,
```
I'd have used alphabetical order criteria here.
`l` coming before `c` causes brain damage.